examples/GPU/example_cg.py

```diff
@@ -60,21 +55,75 @@
 # %%
 # Reconstruct the image using the CG method
 kspace_data = nufft.op(image)  # get the k-space data
-reconstructed_image = cg(nufft, kspace_data)  # reconstruct the image
+reconstructed_image, loss = nufft.cg(
+    kspace_data=kspace_data, num_iter=50, compute_loss=True
+)  # reconstruct the image
+
 
-# %%
 # Display the results
-plt.figure(figsize=(9, 3))
-plt.subplot(1, 3, 1)
+def normalize(img, vmin, vmax):
+    return (img - vmin) / (vmax - vmin)
+
+
+plt.figure(figsize=(15, 10))
+plt.subplot(2, 3, 1)
 plt.title("Original image")
-plt.imshow(abs(image), cmap="gray")
+plt.imshow(normalize(abs(image), abs(image).min(), abs(image).max()), cmap="gray")
+plt.colorbar()
 
-plt.subplot(1, 3, 2)
+plt.subplot(2, 3, 2)
 plt.title("Conjugate gradient")
-plt.imshow(abs(reconstructed_image), cmap="gray")
+plt.imshow(
+    normalize(
+        abs(reconstructed_image),
+        abs(reconstructed_image).min(),
+        abs(reconstructed_image).max(),
+    ),
+    cmap="gray",
+)
+plt.colorbar()
```

Comment:
You dont need to `normalize` your data. Just ensure that vmax and vmin is same in all the cases.
```
    plt.imshow(image, cmap='gray')
    plt.imshow(reconstructed_image, vmax=image.max(), vmin=image.min(), cmap='gray')
```

Do the same for DC Adjoint.